playground/terraform/README.md

```diff
@@ -16,74 +16,118 @@
     specific language governing permissions and limitations
     under the License.
 -->
+# Playground deployment on GCP
 
-# Requirements
+## Prerequisites:
 
-The following items need to be setup for the Playground cluster deployment on GCP:
+### Following items need to be setup for Playground deployment on GCP:
+1. [GCP project](https://cloud.google.com/)
 
-* [GCP account](https://cloud.google.com/)
-* [`gcloud` command-line tool](https://cloud.google.com/sdk/gcloud) and required setup i.e. login
-* [Terraform](https://www.terraform.io/downloads.html) tool
-* [Docker](https://www.docker.com/get-started)
+2. [GCP User account](https://cloud.google.com/appengine/docs/standard/access-control?tab=python) _(Note: You will find the instruction "How to create User account" for your new project)_
 
-# Deployment steps
+***Google Cloud preparation steps:***
+After creating or identified an existing project for deployment, please be sure that your account has at least following privileges:
+   - App Engine Admin
+   - App Engine Creator
+   - Artifact Registry Administrator
+   - Cloud Memorystore Redis Admin
+   - Compute Admin
+   - Create Service Accounts
+   - Kubernetes Engine Admin
+   - Quota Administrator
+   - Role Administrator
+   - Security Admin
+   - Service Account User
+   - Storage Admin
+   - Cloud Datastore Index Admin
 
-## 0. Create GCS bucket for state
+3. [Google Cloud Storage bucket](https://cloud.google.com/storage/docs/creating-buckets) for saving deployment state
 
-```bash
-$ gsutil mb -p ${PROJECT_ID} gs://state-bucket-name
-$ gsutil versioning set on gs://state-bucket-name
-```
+4. DNS name for your Playground deployment instance
 
-## 1. Create new environment
+***Development environment preparation steps:***
 
-To provide information about the terraform backend, run the following commands
+During the Operation system preparation steps, you will need to install Java, Kubernetes, HELM, Docker, Terraform, gcloud CLI
 
-* New environment folder
+* [Java](https://adoptopenjdk.net/)
+* [Kubernetes Command Line Interface](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+* [HELM](https://helm.sh/docs/intro/install/)
+* [Docker](https://docs.docker.com/engine/install/)
+* [Terraform](https://www.terraform.io/downloads)
+* [gcloud CLI](https://cloud.google.com/sdk/docs/install-sdk)
 
-```bash
-mkdir /path/to/beam/playground/terraform/environment/{env-name}
-```
+After Development environment preparation steps completed, navigate to the "beam" folder in your development environment.
```

Comment:
Remove since you explicitly state that gradle commands should be started from repo root

Reply:
fixed